config/development.json

```diff
@@ -1,6 +1,5 @@
 {
   "provider": "http://localhost:8545",
-  "address": "0x9e2d04eef5b16CFfB4328Ddd027B55736407B275",
-  "privateKey": "399b141d0cc2b863b2f514ffe53edc6afc9416d5899da4d9bd2350074c38f1c6",
```

Comment:
Should these be deleted? Just making sure

Reply:
yes, just on dev - because we need to create the wallet from the mnemonic